mixer/cmd/build.go

```diff
@@ -71,6 +73,33 @@ func setWorkers(b *builder.Builder) {
 var buildCmd = &cobra.Command{
 	Use:   "build",
 	Short: "Build various pieces of OS content",
+
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		// Since this function overwrites root's PersistentPreRun, call it manually
+		root := cmd
+		for root.Parent() != nil {
+			root = root.Parent()
+		}
+		if err := root.PersistentPreRunE(cmd, args); err != nil {
+			return err
+		}
+
+		b, err := builder.NewFromConfig(configFile)
+		if err != nil {
+			return err
+		}
+
+		if err := checkFormatMismatch(b); err != nil {
+			return err
+		}
+
+		// Skip format bump check for format-bump commands
+		if marker := cmd.Annotations[bumpMarker]; marker != "" {
```

Comment:
Do you need to actually get the value or can you just check if the key exists?
```go
if _, ok := cmd.Annotations[bumpMarker]; ok {
```

Reply:
yep, that works! I'll update.